src/gui/editors/PianoRoll.cpp

```diff
@@ -3017,8 +3017,8 @@ void PianoRoll::paintEvent(QPaintEvent * pe )
 
 	// set font-size to 80% of key line height
 	QFont f = p.font();
-	f.setPixelSize(m_keyLineHeight * 0.8);
-	p.setFont(f); // font size doesn't change without this for some reason
+	int keyFontSize = m_keyLineHeight * 0.8;
+	p.setFont(adjustedToPixelSize(f, keyFontSize)); // font size doesn't change without this for some reason
```

Comment:
I think the comment can be removed here. The "for some reason" is because that's just how Qt's API works. You get a copy of the current font, can adjust that copy and then have to set the resulting font as the new font. It's not a reference. I guess the API looks like that so that Qt can enforce GUI updates when a font changes.

Reply:
I'll remove it.

Reply:
done with https://github.com/LMMS/lmms/pull/7493/commits/6667544bc65ecc0bda271a55b191b5e2459a368e